e2e/browser/src/components/appContainer/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2021 Inrupt Inc.
```

Comment:
```suggestion
 * Copyright 2022 Inrupt Inc.
```

I'm only gonna comment once, but this is a problem in every file.

It looks like the linting of licenses is not properly setup, I would probably try to re-use the license resource from the root folder and remove the one here altogether.